wayland-client/src/lib.rs

```diff
@@ -42,10 +42,13 @@
 //! managing the newly created object.
 //!
 //! However, implementing all those traits on your own is a lot of (often uninteresting) work. To make this
-//! easier a composition mechanism is provided using the [`delegate_dispatch!`] macro. This way, another
-//! library (such as Smithay's Client Toolkit) can provide generic [`Dispatch`] implementations that you
+//! easier a composition mechanism is provided. Using the [`QueueHandle<State>::make_data()`] method
+//! one is able to create object handlers manually, making it possible to specify `DispatchTo` type diferent
+//! then your `State`.
```

Comment:
```suggestion
//! one is able to create object handlers manually, making it possible to specify `DispatchTo` type different
//! from your `State`.
```

Reply:
Done in ee9140d38bbf6922653196a3fbe1200b175808cc